.github/workflows/ci.yml

```diff
@@ -79,6 +79,26 @@ jobs:
         target:
           - x86_64-apple-darwin
 
+  cc:
+    runs-on: ubuntu-22.04
+    env:
+      TARGET_TRIPLE: x86_64-unknown-linux-gnu
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          submodules: true
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
I don't think there is a need for this line. It's should be fine to use the CI provided `rustc`.